src/apis/otpAuth.go

```diff
@@ -0,0 +1,121 @@
+package apis
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+	verify "github.com/twilio/twilio-go/rest/verify/v2"
+)
+
+type PhoneNumber struct {
+	PhoneNumber string `json:"phone_number"`
+}
+
+type OTP struct {
+	OTP string `json:"otp"`
+}
+
+func (s Server) SendOTP(c *gin.Context) {
+	var phoneNum PhoneNumber
+
+	if err := c.ShouldBind(&phoneNum); err != nil {
+		s.Logger.Error(err)
+		c.JSON(http.StatusBadRequest, errBadRequest.Error())
+		return
+	}
+
+	if err := validatePhoneNumber(phoneNum.PhoneNumber); err != nil {
+		s.Logger.Error(err)
+		c.JSON(http.StatusBadRequest, err.Error())
+		return
+	}
+
+	params := &verify.CreateVerificationParams{}
```

Comment:
`&verify.CreateVerificationParams{}` put it inside the function `fillCreateVerificationParams `